src/llv8.cc

```diff
@@ -549,6 +549,20 @@ std::string Value::Inspect(InspectOptions* options, Error& err) {
 }
 
 
+std::string Value::GetTypeName(InspectOptions* options, Error& err) {
+  Smi smi(this);
+  if (smi.Check()) return smi.Inspect(err);
```

Comment:
Looking at the function, I'd say that it should return something like "(Smi)" here.